### PR TITLE
update oap spark readme

### PR DESCRIPTION
**Pull request by yeyuqiang**

### CODE REVIEW

**Comment by yeyuqiang** [comment]
@zhixingheyi-tian

**Comment by zhixingheyi-tian** [comment]
cc @HongW2019

**Comment by winningsix** [comment]
LGTM

**Comment by zhixingheyi-tian** [comment]
@yeyuqiang 

Would you please add "compile build" developer section ?

**Comment by yeyuqiang** [comment]
> @yeyuqiang
> 
> Would you please add "compile build" developer section ?

update

File: oap-spark/README.md [hunk 1]
```diff
@@ -15,12 +22,22 @@ Before getting start with storage extension with Optane PMem, your machine shoul
...
+
 ### Configuration
 
 To enable rdd cache on Intel Optane PMem, you need add the following configurations:
 ```
 spark.memory.pmem.initial.path [Your Optane PMem paths seperate with comma]
 spark.memory.pmem.initial.size [Your Optane PMem size in GB]
+spark.memory.pmem.usable.ratio [from 0 to 1, 0.85 is recommended]
 spark.yarn.numa.enabled true
 spark.yarn.numa.num [Your numa node number]
```
Comment (HongW2019) [inline oap-spark/README.md:43]: 1. About  Use Optane PMem to cache data, persist(StorageLevel.PMEM_AND_DISK), I fount it difficult to find right codes to replace, so could we offer the location specifically?
2. Just like oap-cache/oap/docs/User-Guide.md , maybe we'd better to put a WebUI picture to tell users when rdd cache enabling at Run K-means benchmark part

File: oap-spark/README.md [hunk 2]
```diff
@@ -15,12 +22,22 @@ Before getting start with storage extension with Optane PMem, your machine shoul
 
 Please refer to documentation at ["Quick Start Guide: Provision Intel® Optane™ DC Persistent Memory"](https://software.intel.com/en-us/articles/quick-start-guide-configure-intel-optane-dc-persistent-memory-on-linux) for detailed to setup Optane PMem with App Direct Mode.
```
Comment (HongW2019) [inline oap-spark/README.md:23]: Here the documentation feels kind of complex, could you please copy Install and configure PMEM steps from ..  like oap-cache/oap/docs/User-Guide.md  responding part?

File: oap-spark/README.md [hunk 3]
```diff
@@ -15,12 +22,22 @@ Before getting start with storage extension with Optane PMem, your machine shoul
...
+
 ### Configuration
 
 To enable rdd cache on Intel Optane PMem, you need add the following configurations:
 ```
 spark.memory.pmem.initial.path [Your Optane PMem paths seperate with comma]
 spark.memory.pmem.initial.size [Your Optane PMem size in GB]
+spark.memory.pmem.usable.ratio [from 0 to 1, 0.85 is recommended]
 spark.yarn.numa.enabled true
 spark.yarn.numa.num [Your numa node number]
```
[the remark this answers is not in the document]
Comment (yeyuqiang) [reply oap-spark/README.md:42]: For q1, the change will happen in user's own spark code. Typically replace the storage level by PMEM_AND_DISK. It's only one line change for the user. I think the doc already told the user which line should be change.
For q2, there's no metrics to indicate the pmem works in webUI. The user may check the usage of pmem to verify,

File: oap-spark/README.md (hunk 2 above)
Comment (yeyuqiang) [reply oap-spark/README.md]: update

File: oap-spark/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (zhixingheyi-tian) [reply oap-spark/README.md:79]: Hi @yeyuqiang 

Can you provide some other evidence if there is no UI screenshot?

[the remark this answers is not in the document]
Comment (yeyuqiang) [reply oap-spark/README.md:79]: No, current no other evidences in RDD Cache with PMEM except check pmem usage

[the remark this answers is not in the document]
Comment (HongW2019) [reply oap-spark/README.md:79]: Hi yuqiang, I have updated the doc about adding details of run K-means,  After you review it ,you can move it to this PR
https://github.com/HongW2019/OAP/blob/doc-0.8.1/oap-spark/README.md

[the remark this answers is not in the document]
Comment (yeyuqiang) [reply oap-spark/README.md:79]: Thanks, Will update with your doc. If no further comments, I will merge it later.

[the remark this answers is not in the document]
Comment (yeyuqiang) [reply oap-spark/README.md:79]: update. @HongW2019

**Pull request merged**